src/annotator/config/test/index-test.js

```diff
@@ -51,7 +51,10 @@ describe('annotator/config/index', function () {
     });
   });
 
-  ['assetRoot', 'subFrameIdentifier'].forEach(function (settingName) {
+  ['appType', 'subFrameIdentifier'].forEach(function (settingName) {
+    // Note, 'assetRoot' should be tested here, but is currently not used in any context
+    // so it needs to be omitted for now.
+    // See https://github.com/hypothesis/client/issues/3474
```

Comment:
To reiterate, `assetRoot` doesn't need to be handled in this context.

Reply:
Roger, thanks.